ci/patch_bazel_windows/patch-bazel

```diff
@@ -0,0 +1,58 @@
+diff --git a/src/BUILD b/src/BUILD
+index a85a9accfd..eeb4318586 100644
+--- a/src/BUILD
++++ b/src/BUILD
+@@ -370,7 +370,7 @@ genrule(
+     outs = ["platforms.zip"],
+     # Removing the WORKSPACE file since local_repository creates it no matter what and leaving
+     # it there would make the zip nondeterministic.
+-    cmd = "zip -qX $@ $$(echo $(SRCS) | xargs -n 1 | grep -v '.*/WORKSPACE$$' | sort | xargs)",
```

Comment:
What was the issue with `choco install zip`? I’d prefer to keep patching Bazel to a minimum and at least at some point that seemed to work. Ideally we would just use `zip` from `msys` but I couldn’t get that to work despite a lot of attempts (just adding msys’ bin directory to PATH broke other stuff).

Reply:
Issue disappeared overnight for no reason that I can tell. Since it had also appeared for no reason, I'll just keep praying for it to stay away. 🤷‍♂️